maat-court-data-api/src/test/java/gov/uk/courtdata/integration/applicant/ApplicantControllerIntegrationTest.java

```diff
@@ -64,18 +67,63 @@ void givenIncorrectRepId_whenGetRepOrderApplicantLinksIsInvoked_thenCorrectError
                 .andExpect(status().isBadRequest());
     }
 
+    @Test
+    void givenValidRequest_whenUpdateRepOrderApplicantLinksIsInvoked_thenUpdateIsSuccess() throws Exception {
+        repOrderRepository.save(TestEntityDataBuilder.getPopulatedRepOrder(REP_ID));
+        repOrderApplicantLinksRepository.saveAndFlush(TestEntityDataBuilder.getRepOrderApplicantLinksEntity());
+        Integer id = repOrderApplicantLinksRepository.findAll().get(0).getId();
+        RepOrderApplicantLinksDTO recordToUpdate = TestModelDataBuilder.getRepOrderApplicantLinksDTO(id);
+        mvc.perform(MockMvcRequestBuilders.put(ENDPOINT_URL + "/rep-order-applicant-links")
+                        .content(objectMapper.writeValueAsString(recordToUpdate))
+                        .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk())
+                .andExpect(content().contentType(MediaType.APPLICATION_JSON))
+                .andExpect(jsonPath("$.id").value(id))
+                .andExpect(jsonPath("$.userModified").value(recordToUpdate.getUserModified()))
+                .andExpect(jsonPath("$.unlinkDate").value(recordToUpdate.getUnlinkDate().toString()));
+    }
+
+    @Test
+    void givenAEmptyContent_whenUpdateRepOrderApplicantLinksIsInvoked_thenCorrectErrorResponseIsReturned() throws Exception {
+        mvc.perform(MockMvcRequestBuilders.put(ENDPOINT_URL + "/rep-order-applicant-links").content("{}")
+                        .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isBadRequest());
+    }
```

Comment:
I'd argue this is already covered by the Controller tests. There's no functional difference since this is just standard validation handled through Spring.